### PR TITLE
tab: include HOMEBREW_VERSION

### PR DESCRIPTION
**Pull request by alyssais**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031).
- [x] Have you successfully run `brew tests` with your changes locally?

-----

Proposed in #1592.

### CODE REVIEW

File: Library/Homebrew/tab.rb [hunk 1]
```diff
@@ -317,6 +319,9 @@ def to_json
   end
 
   def write
+    require 'pry-byebug'
+    binding.pry
```
Comment (jawshooah) [inline Library/Homebrew/tab.rb]: I imagine you'll want to remove this.

Comment (alyssais) [inline Library/Homebrew/tab.rb]: Yeah…

**Comment by MikeMcQuaid** [comment]
It may be worth having a default here rather than `nil`. Thoughts?

**Comment by alyssais** [comment]
I thought about this, but I couldn't think of anything obvious. Any given version string would be incorrect.

**Comment by MikeMcQuaid** [comment]
Yep, seems reasonable. Thanks! @ilovezfs any thoughts?

**Comment by ilovezfs** [comment]
Perhaps fall back to `git describe --tags origin/master` with some designation that it's bogus?

**Comment by MikeMcQuaid** [comment]
@ilovezfs Sorry, I meant of the general PR idea. We will always have HOMEBREW_VERSION set, I was just thinking about how to handle tabs that predate it but I don't think there's a good way (and for what we're using it for: `nil` would be fine)

**Comment by ilovezfs** [comment]
I don't see any actual immediate harm in including it in the receipt, but I'm skeptical we'll ever use it in a trustworthy fashion. I think I'd still prefer a new field, rather than trying to make meaning out of incorrect data and a version string.

Since the problems concern preexisting tabs that will not have had this field at all, I think this would only be usable for some future scenario that has happened yet, unless the mere existence of the homebrew_version field is serving as a proxy for meaning the run time dependencies field is "v2" given that the timeline will roughly correspond to the actual fixes assuming they're imminent, at which point we might as well reduce it to "runtime_dependencies_version=2," since presumably we'll need to bump it again at some point for some currently unknown reason.

**Pull request merged**